### PR TITLE
fix(angular-universal-ssr): not starting

### PR DESCRIPTION
**Pull request by lexasq**
Changed typesctipt version to compatible with angular 10, also removed dependencies for @types/express and @types/node from host-app

### CODE REVIEW

File: angular-universal-ssr/host-app/server.js [hunk 1]
```diff
@@ -3,5 +3,5 @@
 const mainModule = __non_webpack_require__.main;
 const moduleFilename = (mainModule && mainModule.filename) || '';
 if (moduleFilename === __filename || moduleFilename.includes('iisnode')) {
-  import('./server-bootstrap').then(x => x.run());
+  import('./server-bootstrap.ts').then(x => x.run());
```
Comment (PeaceDate) [inline angular-universal-ssr/host-app/server.js:6]: host-app has errors 
Screenshot 2023-01-12 at 19 31 34

Please take a look)

Comment (PeaceDate) [inline angular-universal-ssr/host-app/server.js:6]: After clicking on `Federation lazy module`(http://localhost:4000/federation) get error
Screenshot 2023-01-12 at 19 31 19

Also please take a look

Comment (lexasq) [reply angular-universal-ssr/host-app/server.js:6]: Where did you get this port from? Asking because this app seems to be using only 4000 and 5000
image

Comment (lexasq) [reply angular-universal-ssr/host-app/server.js:6]: Have you made yarn install for both apps and root?
image

Comment (lexasq) [reply angular-universal-ssr/host-app/server.js:6]: Mine local works, that's why I'm asking
image

**Review by PeaceDate — DISMISSED** [review]

**Pull request merged**